### PR TITLE
ci: comment on end of plans

### PR DESCRIPTION
**Pull request by CalvinRodo**
Writes a comment when plans are done

### CODE REVIEW

File: .github/workflows/pull-request.yml [hunk 1]
```diff
@@ -278,3 +278,21 @@ jobs:
...
+    if: always()
+    needs: [tg-plan-dependents]
+    runs-on: ubuntu-latest
+    steps:
+
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - uses: actions/github-script@v4
+        env:
```
Comment (patheard) [inline .github/workflows/pull-request.yml:292]: We can smoke this `env` block

**Review by patheard — CHANGES_REQUESTED** [review]
Comment about `env` block

File: .github/workflows/pull-request.yml (hunk 1 above)
Comment (CalvinRodo) [reply .github/workflows/pull-request.yml]: done.

**Review by patheard — APPROVED** [review]
👍🏻

**Pull request merged**